### DISCUSSION TITLE
Untrusted Host "<hostname>" exception when upgrading 1.13

### DISCUSSION THREAD

**Discussion by irshadsm** [kimai/kimai#2464] [Questions & Answers]
I have kimai installed in openshift, but when accessing the routes, i am getting Untrusted Host "<hostname>"  exception. Could you please help me to resolve it?
Symfony\Component\HttpKernel\Exception\
BadRequestHttpException

Regards,
Irshad

**Comment by kevinpapst** [comment]
https://www.kimai.org/documentation/webserver-configuration.html#reverse-proxy ?
